src/sentry/api/bases/organization_events.py

```diff
@@ -130,9 +130,6 @@ def get_snuba_dataclass(
                         has_global_views,
                         len(params.projects) <= 1,
                         fetching_replay_data,
-                        # If a developer can view issues of a project they do not belong to
-                        # via open membership, we will also allow the endpoint to return events for it
-                        organization.flags.allow_joinleave,
```

Comment:
This was added to my original PR.

This has been moved down below:
<img width="521" alt="image" src="https://github.com/user-attachments/assets/9829d7fe-1c24-4579-a516-734d56e336a6">